src/server-code/main-server.js

```diff
@@ -220,7 +193,6 @@ export default function runMainServer(serverFalcorModel) {
       );
     });
   }
-
```

Comment:
No reason to remove this newline is there? Just decreases readability if anything